docs/api-examples/hubspot.mdx

```diff
@@ -32,7 +32,30 @@ HubSpot's APIs are available as [Open API Specifications](https://api.hubspot.co
 superface prepare https://api.hubspot.com/api-catalog-public/v1/apis/crm/v3/objects/companies "hubspot"
 ```
 
-Once the documentation has been indexed, you can create a profile for the use case you want to achieve, which in this case is to get a list of companies in the CRM.
+Once the documentation has been indexed, you should check that the API `baseUrl` and `securityScheme` are correct in the `hubspot.provider.json` file. You can copy the example below if needed:
+
+```json title="hubspot.provider.json
```

Comment:
We fixed the trailing slashes in engine but this is also useful, I think. Thanks 🙏